README.md

```diff
@@ -1,5 +1,10 @@
 # Variation Normalizer Manuscript
 
+### Results
+![Patient Matching with GENIE](./analysis/genie/variant_analysis/genie_patient_matching.png)
+
+Variant normalization allows patient data from AACR Project GENIE to be matched to normalized variants in the CIViC, MOA, and ClinVar knowledgebases.
```

Comment:
Move this after the data in repo / s3 section. And have the text go before the image and state like "as shown in the following figure"